app/controllers/cards_controller.rb

```diff
@@ -47,4 +50,9 @@ def card_params
   def find_card
     @card = Card.find(params[:id])
   end
+
+  def not_authenticated
+    flash[:alert] = 'У вас нету доступа до этой страницы! Войдите в свой профиль или зарегистрируйтесь'
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.